frontend/src/scenes/trends/LineGraph.js

```diff
@@ -185,7 +188,7 @@ export class LineGraph extends Component {
     }
 }
 LineGraph.propTypes = {
-    datasets: PropTypes.arrayOf(PropTypes.shape({ label: PropTypes.string, count: PropTypes.number })).isRequired,
+    datasets: PropTypes.arrayOf(PropTypes.shape({ label: PropTypes.string, count: PropTypes.any })).isRequired,
```

Comment:
Why is number any now?

Reply:
Restructured the labels so this is number again!